src/graphql/schema/getMutationGqlType.ts

```diff
@@ -17,7 +17,7 @@ export default getMutationGqlType
  *
  * @private
  */
-function createMutationGqlType (buildToken: BuildToken): GraphQLObjectType<mixed> | undefined {
+function createMutationGqlType (buildToken: BuildToken, disableDefaultMutations?: boolean): GraphQLObjectType<mixed> | undefined {
   const { inventory } = buildToken
```

Comment:
Here you’d take out the second argument, and rewrite the above line to:

```js
const { inventory, options } = buildToken
```

…and below just change `disableDefaultMutations` to `options.disableDefaultMutations`.